software/database/pom.xml

```diff
@@ -107,18 +107,10 @@
             <artifactId>groovy-all</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.beust</groupId>
-            <artifactId>jcommander</artifactId>
-        </dependency>
-        <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.google.code.findbugs</groupId>
-            <artifactId>jsr305</artifactId>
```

Comment:
i agree, with richard (and maven recommendation) that projects declare their compile-time deps -- will restore.  somehow missed the fact this was being pulled in from other projects.